Docs/user_guide/coordinate_systems.md

```diff
@@ -0,0 +1,269 @@
+column-the translation vector-to the right, and a $$1$$ in the lower right
+corner. Additionally, all vectors have to be written as homogeneous
+coordinates, which means that a $$1$$ is augmented at the end.
+
+$$
+\begin{pmatrix} x_1 \\ x_2 \\ x_3 \\ 1 \end{pmatrix} =
+\begin{pmatrix} A_{11} & A_{12} & A_{13} & t_1 \\ A_{21} & A_{22} & A_{23} & t_2 \\
+                A_{31} & A_{32} & A_{33} & t_3 \\ 0 & 0 & 0 & 1 \end{pmatrix}
+\begin{pmatrix} i \\ j \\ k \\ 1 \end{pmatrix}
+$$
+
+Depending on the used anatomical space (LPS or RAS) the $$4 \times 4$$ matrix
+is called **IJKtoLPS**- or **IJKtoRAS**-matrix, because it represents the
+transformation from IJK to LPS or RAS.
+
+## 2D example or calculating an *IJtoLS*-matrix
+
+The following figure shows the anatomical space with an L(P)S basis on the
+left and the corresponding image coordinates on the right.
+
+![IJtoLS](../_static/images/IJtoLS.png)
+
+The origin (the coordinates of the first pixel in anatomical space) is
+(50, 300) mm and the spacing (the distance between two pixels) is
+(50, 50) mm.
+
+As this is a 2D example $$\mathbf{A}$$ is a $$2 \times 2$$ matrix and
+$$\vec{t}$$ a $$2 \times 1$$ vector. Therefore, the equation of the affine
+transformation is:
+
+$$
+\begin{pmatrix} L \\ S \\ 1 \end{pmatrix} =
+\begin{pmatrix} A_{11} & A_{12} & t_1 \\ A_{21} & A_{22} & t_2 \\ 0 & 0 & 1 \end{pmatrix}
+\begin{pmatrix} i \\ j \\ 1 \end{pmatrix}
+$$
+
+By multiplying the **IJtoLS**-matrix and the vector of the right side, the
+following product will be obtained:
+
+![matrix_multiplication](../_static/images/matrix_multiplication.png)
+
+The last equation and the matrix product show that a total of 6 unknown
+variables $$(A_{11}, A_{12}, A_{21}, A_{22}, t_1, t_2)$$ have to be
+determined. The knowledge of origin and spacing however allows the following
+relations between image and anatomical space:
+
+$$
+\begin{pmatrix} L \\ S \end{pmatrix} \equiv \begin{pmatrix} i \\ j \end{pmatrix} \qquad
+\begin{pmatrix} 50 \\ 300 \end{pmatrix} \equiv \begin{pmatrix} 0 \\ 0 \end{pmatrix} \qquad
+\begin{pmatrix} 100 \\ 300 \end{pmatrix} \equiv \begin{pmatrix} 1 \\ 0 \end{pmatrix} \qquad
+\begin{pmatrix} 50 \\ 250 \end{pmatrix} \equiv \begin{pmatrix} 0 \\ 1 \end{pmatrix} \qquad \dots
+$$
+
+Thus, at least six equations can be derived:
+
+$$
+50 &= A_{11} \cdot 0 + A_{12} \cdot 0 + t_1 \cdot 1
+300 &= A_{21} \cdot 0 + A_{22} \cdot 0 + t_2 \cdot 1
+100 &= A_{11} \cdot 1 + A_{12} \cdot 0 + t_1 \cdot 1
+300 &= A_{21} \cdot 1 + A_{22} \cdot 0 + t_2 \cdot 1
+50 &= A_{11} \cdot 0 + A_{12} \cdot 1 + t_1 \cdot 1
+250 &= A_{21} \cdot 0 + A_{22} \cdot 1 + t_2 \cdot 1
+$$
+
+As mentioned above, the translation $$\vec{t}$$ contains the information
+about the geometric position of the first pixel and is therefore equivalent
+to the origin. This result is also confirmed by the first equations.
+
+The solution of the other equations leads to the following **IJtoLS**-matrix:
+
+$$
```

Comment:
To support this syntax, we should set `myst_dmath_allow_labels`[^1] in `Docs/conf.py`[^2]

[^1]: https://myst-parser.readthedocs.io/en/latest/syntax/math.html
[^2]: https://github.com/Slicer/Slicer/blob/main/Docs/conf.py

Reply:
@jcfr I am fine if we use the [math role and directive](https://myst-parser.readthedocs.io/en/latest/syntax/math.html#math-role-and-directive) and avoid installing additional extensions if that is preferred. Please let me know what is the preference.

Reply:
To also support inline formula using `$$`, I suggest we enable the myst extension. This can be done by adding `"dollarmath"` to `myst_enable_extensions` in `conf.py`.

Since it is a built-in myst extension, there is no need to install any additional package.

Reply:
Consider also setting `myst_dmath_allow_labels` to `true` so that equation can be associated with a label and referenced within the document.

Reply:
Done in [17cea93](https://github.com/Slicer/Slicer/commit/17cea93d3bc6b3279333b3f3a71728394a55bd7c).